lib/gui/app.js

```diff
@@ -100,14 +99,12 @@ app.run((ErrorService, SelectionStateModel) => {
   });
 
   const currentReleaseType = release.getReleaseType(currentVersion);
+  const enableUpdates = settings.get('enableUpdates');
```

Comment:
Perhaps `updatesEnabled` would be better, as `enableUpdates` could be misinterpreted as a verb (command) ?